### PR TITLE
Use status index during status validation

### PR DESCRIPTION
**Pull request by eshryane**
Use the status index to find status of children, don't load entire objects

### CODE REVIEW

**Comment by eshryane** [comment]
Integration tests have passed

File: whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumSpec.groovy [hunk 1]
```diff
@@ -1449,8 +1449,8 @@ class InetnumSpec extends BaseQueryUpdateSpec {
         ack.countErrorWarnInfo(2, 0, 1)
         ack.errors.any { it.operation == "Create" && it.key == "[inetnum] 62.59.192.2 - 62.59.192.30" }
         ack.errorMessagesFor("Create", "[inetnum] 62.59.192.2 - 62.59.192.30") ==
-                ["This range overlaps with 62.59.192.0 - 62.59.192.7",
-                "Status ASSIGNED PA not allowed when more specific object '62.59.192.8 - 62.59.192.15' has status ALLOCATED PA"]
+                ["Status ASSIGNED PA not allowed when more specific object '62.59.192.8 - 62.59.192.15' has status ALLOCATED PA",
```
Comment (eshryane) [inline whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumSpec.groovy:1452]: FYI some messages reversed order when I renamed / moved the validator classes

File: whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcStatusDao.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            objectId);
+    }
+
+    @Override
+    public Map<Integer, CIString> getStatus(final List<Integer> objectIds) {
+        final Map<Integer, CIString> results = Maps.newHashMap();
+        final Map<String, Object> params = Maps.newHashMap();
+
+        // partition object ids to keep inside the max_packet_size limit for the IN clause
+        for (List<Integer> partition : Lists.partition(objectIds, 10_000)) {
```
Comment (sbusk) [inline whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcStatusDao.java:46]: Use constant for the magic number.

**Review by sbusk — APPROVED** [review]

File: whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcStatusDao.java (hunk 2 above)
Comment (eshryane) [reply whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcStatusDao.java]: Done, thanks!

**Pull request merged**